docs/gradle-getting-started.md

```diff
@@ -224,40 +231,42 @@ In this step you'll learn how to propagate a local change to the application thr
 Modify `build.gradle` to include a dependency for `jscience` in the `dependencies` section like this:
 
 ```groovy
-:::>> file.append build.gradle#16
+:::>> file.append build.gradle#30
 compile "org.jscience:jscience:4.3.1"
 ```
 
 Modify `src/main/java/Main.java` so that it imports this library at the start, by including the following imports:
```

Comment:
This looks to be the wrong path now.  `src/main/java/com/example/heroku/HerokuApplication.java` ?

Reply:
good catch

Reply:
Fixed in ca9a552